### PR TITLE
webserver/site: add button to revisit settings page from dexsettings

### LINKED ISSUE #2545: ui: DEX server edit screen doesn't have a close/back button
The update DEX server dialog can not be closed the same way as other dialogs, the only way to get back to the Settings UI is to use the browser's back button.

image

### PR DESCRIPTION
**Pull request by ukane-philemon**
Closes #2545

Screenshot 2023-10-02 at 7 18 12 AM

### CODE REVIEW

**Review by JoeGruffins — APPROVED** [review]

File: client/webserver/site/src/css/icons.scss [hunk 1]
```diff
@@ -110,6 +110,11 @@
   transform: rotate(180deg);
 }
 
+.ico-wide-headed-left-arrow::before {
+  content: "\e919";
```
Comment (buck54321) [inline client/webserver/site/src/css/icons.scss:114]: You forgot to add the `<glyph>` to the SVG file

File: client/webserver/site/src/html/dexsettings.tmpl [hunk 2]
```diff
@@ -2,6 +2,7 @@
 {{template "top" .}}
 {{$passwordIsCached := .UserInfo.PasswordIsCached}}
 <div id="main" data-handler="dexsettings" data-host="{{.Exchange.Host}}" class="text-center py-5 overflow-y-auto">
+  <div class="go-back-to-settings"><span id="goBackToSettings" class="ico-wide-headed-left-arrow fs24"></span></div>
   <span class="settings-gear ico-settings"></span>
```
Comment (buck54321) [inline client/webserver/site/src/html/dexsettings.tmpl:6]: Please consider these changes and then deleting the `go-back-to-settings` class.
```suggestion
<div id="main" data-handler="dexsettings" data-host="{{.Exchange.Host}}" class="flex-center flex-column align-items-stretch py-5 mx-auto">
  <div id="goBackToSettings" class="d-flex justify-content-start align-items-center pointer hoverbg py-2">
    <span class="ico-wide-headed-left-arrow fs24 lh1"></span>
  </div>
  <div class="flex-center pb-2"><span class="settings-gear ico-settings"></span></div>
```

File: client/webserver/site/src/js/dexsettings.ts [hunk 3]
```diff
@@ -36,6 +36,7 @@ export default class DexSettingsPage extends BasePage {
     Doc.bind(page.updateCertBtn, 'click', () => page.certFileInput.click())
     Doc.bind(page.updateHostBtn, 'click', () => this.prepareUpdateHost())
     Doc.bind(page.certFileInput, 'change', () => this.onCertFileChange())
+    Doc.bind(page.goBackToSettings, 'click', () => window.location.assign('/settings'))
```
Comment (buck54321) [inline client/webserver/site/src/js/dexsettings.ts:39]: **dexc** is a single-page app.

```Typescript
Doc.bind(page.goBackToSettings, 'click', () => app().loadPage('settings'))
```

**Review by buck54321 — CHANGES_REQUESTED** [review]

File: client/webserver/site/src/css/icons.scss (hunk 1 above)
Comment (ukane-philemon) [reply client/webserver/site/src/css/icons.scss:114]: Dang. I've updated the SVG file and re-generated the font files.

File: client/webserver/site/src/html/dexsettings.tmpl (hunk 2 above)
Comment (ukane-philemon) [reply client/webserver/site/src/html/dexsettings.tmpl]: Thanks for the suggestion. I've updated this file with minor modification to your suggestion.

**Review by buck54321 — APPROVED** [review]

**Pull request merged**